src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -1265,14 +1078,7 @@ class SaagieClient {
                 }
             }
             return exportJob
-        } catch (InvalidUserDataException invalidUserDataException) {
-            throw invalidUserDataException
-        } catch (GradleException stopActionException) {
-            throw stopActionException
-        } catch (Exception exception) {
-            logger.error('Unknown error in getJobAndJobVersionDetailToExport')
-            throw exception
-        }
+        },'Unknown error in getJobAndJobVersionDetailToExport') as ExportJobs
```

Comment:
getProjectJobsByNameAndId

Reply:
![image](https://user-images.githubusercontent.com/57661415/85867212-743d4e00-b7c0-11ea-8029-3bb661c38db2.png)

Reply:
One more time i don t know why github is showing getProjectJobsByNameAndId instead of getJobAndJobVersionDetailToExportV1